tests/integration-tests/tests/schedulers/test_slurm/test_fast_capacity_failover/overrides.sh

```diff
@@ -32,4 +32,24 @@ def run_instances(region, boto3_config, **run_instances_kwargs):
     else:
         ec2_client = boto3.client("ec2", region_name=region, config=boto3_config)
         return ec2_client.run_instances(**run_instances_kwargs)
+
+
+def create_fleet(region, boto3_config, **create_fleet_kwargs):
+    lt_spec = create_fleet_kwargs.get("LaunchTemplateConfigs", [])[0].get("LaunchTemplateSpecification")
+    if "ice-cr-multiple" in lt_spec.get("LaunchTemplateName"):
```

Comment:
handle the case LaunchTemplateSpecification is None, e.g.

```
lt_name = create_fleet_kwargs.get("LaunchTemplateConfigs", [])[0].get("LaunchTemplateSpecification", {}).get("LaunchTemplateName")
```

Reply:
Done